Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -272,6 +272,16 @@ public extension Sequence {
     func filter(by keyPath: KeyPath<Element, Bool>) -> [Element] {
         return filter { $0[keyPath: keyPath] }
     }
+
+    /// SwifterSwift: Returns the first element of the sequence with having property by given key path equal to given `value`.
+    ///
+    /// - Parameters:
+    ///   - keyPath: The `KeyPath` of property for `Element` to compare.
+    ///   - value: The value to compare with `Element` property
+    /// - Returns: The first element of the collection that
+    func first<T: Equatable>(where keyPath: KeyPath<Element, T>, equal value: T) -> Element? {
```

Comment:
Calls to this function will read better if you use `equals`, since you're checking for singular values.
```suggestion
    func first<T: Equatable>(where keyPath: KeyPath<Element, T>, equals value: T) -> Element? {
```

Reply:
Nice point, I agree.